crates/biome_analyze/CONTRIBUTING.md

```diff
@@ -649,6 +659,16 @@ This is what the `Deserializable` keyword in the `#[derive]` statements above di
 It's a so-called derive macros, which generates the implementation for the `Deserializable` trait
 for you.
 
+The rule's options type have also to implement `biome_deserialize::Merge`.
```

Comment:
We must add a paragraph that says that all options must be wrapped in `Option<T>`